rmw_fastrtps_cpp/src/functions.cpp

```diff
@@ -507,7 +507,7 @@ class WriterInfo : public ReaderListener
   {
     WriterProxyData proxyData;
     if (change->kind == ALIVE) {
-      CDRMessage_t tempMsg;
+      CDRMessage_t tempMsg(0);
```

Comment:
I'll approve it, yeah.  Can you please open an issue against Fast-RTPS so we can address this better in the future?  Thanks.